contrib/opbot/botmd/commands.go

```diff
@@ -336,6 +336,22 @@ func (b *Bot) rfqRefund() *slacker.CommandDefinition {
 				return
 			}
 
+			isRelayed, err := fastBridgeContract.BridgeRelays(nil, [32]byte(common.Hex2Bytes(rawRequest.TxID)))
```

Comment:
`fastBridgeContract` is the contract on the origin chain, whereas we need to check if the relay happened on the destination.

I guess one of the possible solutions would involve adjusting `makeFastBridge` to make both origin and destination instances.